package-manifest.toml

```diff
@@ -716,8 +716,8 @@ only_for_targets.image = "standard"
 # the other `source.*` keys.
 source.type = "prebuilt"
 source.repo = "dendrite"
-source.commit = "f3810e7bc1f0d746b5e95b3aaff32e52b02dfdfa"
-source.sha256 = "c1506f6f818327523e6ff3102432a2038d319338b883235664b34f9132ff676a"
+source.commit = "f79076d4630895a0177839ea041c355e2589b9c1"
+source.sha256 = "7dbd752327234d6321ae9b06fb1eb72874bd3bcf0122ee545f60fe2579d9746c"
```

Comment:
TODO: settle these hashes when https://github.com/oxidecomputer/dendrite/pull/1031 is settled.